src/pages/resultsView/querySummary/QuerySummary.tsx

```diff
@@ -150,6 +154,18 @@ export default class QuerySummary extends React.Component<{ routingStore:Extende
         return this.props.store.genesInvalid || this.props.store.isQueryInvalid;
     }
 
+    @computed get getQueryStore() {
+        let queryStore = createQueryStore(getBrowserWindow().routingStore.query);
+        if (this.props.store.selectedSampleIdsForQueryButton) {
+            queryStore.caseIds = this.props.store.selectedSampleIdsForQueryButton;
```

Comment:
i don't think we should store the info for the query in the resultsViewStore.  It's a kind of transient state.  Like a message.